### DISCUSSION TITLE
[Feature] oled black for mobile app

### DISCUSSION THREAD

**Discussion by nightfury05** [immich-app/immich#14315] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

most of the phones this days have oled screen and oled pitch black looks so good in dark mode. i dont think it would be hard to add just option to change to oled black. i dont think how usefull this would be on web but nice to have for sure

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#13698
